client/src/services/resourceService.js

```diff
@@ -16,6 +16,13 @@ export default class ResourceService {
 		return [];
 	}
 
+	async getOne(id) {
+		const res = await this.fetch(`${ResourceService.ENDPOINT}/${id}`);
+		if (res.ok) {
```

Comment:
This function currently does not handle the case if the response status is not ok or provide any feedback. 

Suggestion: Returning an error or throwing an exception would make it easier to handle errors in the calling code.